docs/developing-locally.rst

```diff
@@ -141,15 +141,23 @@ In production, we have Mailgun_ configured to have your back!
 Celery
 ------
 
-If the project is configured to use Celery as a task scheduler then by default tasks are set to run on the main thread
-when developing locally. If you have the appropriate setup on your local machine then set the following
-in ``config/settings/local.py``::
+If the project is configured to use Celery as a task scheduler then, by default, tasks are set to run on the main thread when developing locally instead of getting sent to a broker. However, if you have Redis setup on your local machine, you can set the following in ``config/settings/local.py``::
```

Comment:
I was also confused by the original wording of this, as I wasn't sure what "appropriate setup" means, and whether I had the appropriate setup. After doing some research, I think it means having a broker setup, so I reworded it to that.

Reply:
I think that's clearer, yes 👍🏻